vortex-datafusion/src/persistent/format.rs

```diff
@@ -117,10 +123,12 @@ impl VortexFormat {
     /// Create a new instance of the [`VortexFormat`].
     pub fn new(context: ContextRef) -> Self {
         let opts = VortexFormatOptions::default();
+        let io_dispatcher = IoDispatcher::new_tokio(opts.dispatch_threads);
```

Comment:
Is this shared across all files? Or will there be 100 of these for clickbench?

Reply:
One per table, like the layout cache